config/locales/en.yml

```diff
@@ -292,7 +292,7 @@ en:
     recovery_code_html: 'You can use a valid  <a href="https://guides.rubygems.org/setting-up-multifactor-authentication/#using-recovery-codes-and-re-setup-a-previously-enabled-mfa", class="t-link">recovery code</a> if you have lost access to your MFA device.'
     incorrect_otp: Your OTP code is incorrect.
     otp_code: OTP code
-    require_mfa_disabled: Your multifactor authentication has been enabled. You have to disable it first.
+    require_mfa_disabled: Your multifactor authentication has been enabled. To reconfigure MFA, you'll have to disable it first.
```

Comment:
:information_source: We should not use MFA - https://github.com/rubygems/rubygems.org/pull/2685 + we should use `multi-factor` instead of `multifactor` (per https://github.com/rubygems/rubygems.org/pull/3093/commits/0b598ce695aa79665bef8be8cb367bc50b570e9c).